tests/python/pants_test/backend/python/tasks/test_python_distribution_integration.py

```diff
@@ -46,7 +44,8 @@ def test_pants_binary(self):
       output = subprocess.check_output(pex).decode('utf-8')
       self._assert_native_greeting(output)
       # Check that we have exact one wheel output
-      self.assertEqual(len(glob.glob(wheel_glob)), 1)
+      single_wheel_output = assert_single_element(glob.glob(os.path.join(tmp_dir, '*.whl')))
+      self.assertTrue(os.path.basename(single_wheel_output).startswith('fasthello-1.0.0+'))
```

Comment:
If you use `assertRegexpMatches` you'll get a better error message.

Reply:
Done in 4456105!